src/arangodb/storage.py

```
@@ -638,33 +646,85 @@ def update_item(self, item_id: str, data: Dict[str, Any]) -> bool:
+            # Insert into documents collection
+            if self._client is None:
+                raise ValueError("ArangoDB client not connected")
+            result = self._client.insert_document("documents", doc_dict)
+            
+            # Return full document ID
+            return f"documents/{result['_key']}"
+            
+        except Exception as e:
+            error_msg = f"Failed to create document: {e}"
+            self.logger.error(error_msg)
+            raise ArangoOperationError(error_msg)
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)